docs/user/02-configuration.md

```diff
@@ -25,6 +25,10 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 - [Minimal CR](https://github.com/kyma-project/nats-manager/blob/main/config/samples/minimal.yaml#L1)
 - [Full spec CR](https://github.com/kyma-project/nats-manager/blob/main/config/samples/nats-full-spec.yaml#L1)
 
+## High availability
+
+NATS Servers are set up across different Availability Zones for uninterrupted operation and uptime. To achieve this, a minimum of three Kubernetes nodes must be distributed across at-least 3 Availability zones. Failure to meet this requirement will result in NATS Servers being scheduled within the same Availability zone, compromising high availability.
```

Comment:
```suggestion
NATS Servers are set up across different availability zones for uninterrupted operation and uptime. To achieve this, a minimum of three Kubernetes Nodes must be distributed across at least three availability zones. If this requirement is not met, high availability is compromised because results in NATS Servers are scheduled within the same availability zone.
```

Who distributes the Nodes? Is it the reader, or does that happen automatically? And if automatically, how can the requirement be not met?

Reply:
it happens automatically by k8s. If the k8s cluster do not have nodes in 3 different AVs then the requirement is not met.

Reply:
updated!